assets/js/googlesitekit-datastore-site.js

```diff
@@ -6,6 +6,7 @@
  * Internal dependencies
  */
 import Data from 'googlesitekit-data';
-import { registerOn } from 'assets/js/googlesitekit/datastore/site';
+import store, { STORE_NAME } from 'assets/js/googlesitekit/datastore/site';
 
-registerOn( Data.registry );
+// Register this store on the global registry.
+Data.registerStore( STORE_NAME, store );
```

Comment:
Per the JS guidelines doc, functionality like this should happen in the module itself. All the wrapper file should ever do is expose the API on a global if necessary or (in this case) really just import the file so that it gets executed. Essentially this file would become a single-line item.

Reply:
Hmmm, registering the store in the file that defines it is quite the side effect, and means this file will only be doing an import with a side-effect. I liked keeping the side-effect imports at the top-most level. We can make that change, but I like the idea that side-effects like attaching to globals or calling registration functions happen in the entrypoints.